fedbiomed/common/message.py

```diff
@@ -402,27 +461,13 @@ class KeyReply(InnerRequestReply, RequiresProtocolVersion):
     Attributes:
         public_key: public key of replying node
         secagg_id: unique ID of this secagg context element
-        command: Command string
 
     Raises:
         FedbiomedMessageError: triggered if message's fields validation failed
     """
+
     public_key: bytes
     secagg_id: str
-    command: str
-
-# Example of  of one-wway (not request-reply) inner message
-#
-# @catch_dataclass_exception
-# @dataclass
-# class DummyInner(InnerMessage, RequiresProtocolVersion):
-#     """Dummy example of one-wway (not request-reply) inner message
-#     """
-#     dummy: str     # Temporary dummy payload
-#     command: str
-
-
-# --- Node <=> Researcher messages ----------------------------------------------
```

Comment:
Misc: even if we suppress the example of one-way n2n message we need to keep the separation of node <=> researcher messages

```
# --- Node <=> Researcher messages ----------------------------------------------
```

Reply:
Ok. However, this is just a comment. I think that  if you want to separate these different messages the correct way to do it 
1 - create sperate modules
2 - make them inherit from class that is named explicitly, for example -> `overlay -> NodeToNode`